istioctl/cmd/testdata/authz/converter/policy-destination-labels.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ClusterRbacConfig
+metadata:
+  name: default
+spec:
+  mode: 'ON_WITH_INCLUSION'
+  inclusion:
+    namespaces: ["default"]
+---
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ServiceRole
+metadata:
+  name: role
+  namespace: default
+spec:
+  rules:
+  - services: ["productpage.default.svc.cluster.local"]
+    methods: ["GET"]
+    constraints:
+    - key: "destination.labels[env]"
+      values:
+      - "env1"
+      - "env2"
+    - key: "destination.labels[tag]"
+      values:
+      - "tag1"
+      - "tag2"
+    - key: "destination.labels[machine]"
```

Comment:
Do we still support destination.labels at attribute key in v1beta1?

Reply:
No, the tool now converts it to use the workload selector. See `istioctl/cmd/testdata/authz/converter/policy-destination-labels.yaml` and `istioctl/cmd/testdata/authz/converter/policy-destination-labels-want.yaml` for example.